### DISCUSSION TITLE
托運行李須禁用 AirTag？德國航空公司反悔不禁用了

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#783] [General]
airtag真的很方便，應該說蘋果生態系除了沒有電池的配件外，幾乎都有定位追蹤功能，操作介面也很簡單，對於忘東忘西的人真的是最佳選擇
[文章連結](https://www.kocpc.com.tw/archives/463734)](https://www.kocpc.com.tw/archives/463734)

**Comment by Yunhans** [comment]
但我現在看不到airtag的電池電量了
為什麼要改掉啊蘋果

**Comment by Pietra1226** [comment]
我自己有買airtag，有次錢包丟在餐廳馬上就找到了

**Comment by Doreen331** [comment]
airtag雖然方便，但是也有危險的地方，可能會有有心人士利用airtag追蹤別人（不論駭進帳號或是利用實體的airtag），但大家好像對這方面的疑慮沒有很關心，畢竟一堆人都在下載冰棒

**Comment by Kotori0501** [comment]
感謝分享
好奇問個這只是用於iPhone嗎？
我也經常忘東忘西希望可以引進到Android，這樣對我來說便利很多

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
真的是一樣很方便的工具 對於我這種忘東忘西的個姓 真的幫了我不少

**Comment by dinosaur1009** [comment]
感覺airtag可以再壓低價格....可以越來越普及

**Comment by StevenWanglolz** [comment]
我不台常掉東西，而且airtag的價格不太親民，所以應該震時不會考慮...
